frameworks/elastic/docs/limitations.md

```diff
@@ -15,13 +15,19 @@ The maximum number of deployable nodes is constrained by the DC/OS cluster's res
 
 During deployment and upgrades, the `serial` strategy does not wait for the Elastic service to reach green before proceeding to the next node.
 
+## Toggling TLS requires doing a full-cluster restart
+
+This is an [Elasticsearch limitation](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html). Since the service's default update strategy is a rolling upgrade (`serial`) changing its configuration to enable (or disable) transport encryption will result in nodes configured with TLS not being able to communicate with nodes configured with unencrypted networking (and vice-versa). You must do a full-cluster restart (`parallel` update strategy). Make sure to have backups and plan for some downtime.
+
+Afterwards, you can set the update strategy back to `serial` for future updates.
```

Comment:
Is this a "can", or a "should"?

Reply:
Hm, good question. It is not absolutely required, but a good idea in most cases. I used the same wording as in the [X-Pack docs](https://github.com/mesosphere/dcos-commons/blob/master/frameworks/elastic/docs/elastic-x-pack.md) fwiw.

Should we change both to "should"?

Reply:
If "serial" is our recommended mode of operation, then "can" is a bit loose.

I'm OK to leave it as "can" if X-Pack is considered the reference. We could ask @loren or @benclarkwood for their input.